src/tools/ipv4-range-expander/ipv4-range-expander.service.ts

```diff
@@ -0,0 +1,70 @@
+import { convertBase } from '../integer-base-converter/integer-base-converter.model';
+import { ipv4ToInt } from '../ipv4-address-converter/ipv4-address-converter.service';
+class Ipv4RangeExpanderResult {
+  oldSize?: number;
+  newStart?: string;
+  newEnd?: string;
+  newCidr?: string;
+  newSize?: number;
+}
```

Comment:
I not fan of mixing paradigms, can you create a type (in a .type.ts file) instead of classe 🙏🏻